src/app/gauge/gauge.component.html

```diff
@@ -1,35 +1,34 @@
-<section class="angular-gauge" [class.light]="lightTheme">
-  <svg class="info" [attr.viewBox]="viewBox" xmlns="http://www.w3.org/2000/svg">
-    <circle *ngIf="light"
-      class="red-light"
+<section class="ng-gauge" [class.light]="lightTheme" >
+  <svg class="info" [attr.viewBox]="viewBox" xmlns="http://www.w3.org/2000/svg" [attr.width]="Configuration.WIDTH">
```

Comment:
Let's move the width as an inlined style of the `section` element, i.e.
```
<section class="ng-gauge" [class.light]="lightTheme" [style.width]="config.WIDTH + 'px'">
```
since this is what we desire.